CalculatorExercises/1-CalculatorBasic/Calculator.java

```diff
@@ -0,0 +1,43 @@
+import java.util.*;
+import java.lang.*;
+
+public class Calculator {
+   
+    private double result;
+
+    public double getResult() {
+        return result;
+    }
+
+    public Calculator () {
+        this.result = 0;
+    }
+
+    public void calculate (String input) {
+        try {
+                switch (input.charAt(0)) {
+                case '+':
+                    result += Double.valueOf(input.substring(1)); 
+                    break;
+                case '-':
+                    result -= Double.valueOf(input.substring(1));
+                    break;
+                case '/':
+                    result /= Double.valueOf(input.substring(1));
+                    break;
+                case '*':
+                    result *= Double.valueOf(input.substring(1));
+                    break;
+                default : 
+                    System.out.println("Zle polecenie");
```

Comment:
throw new IllegalArgumentException(); here
https://docs.oracle.com/javase/7/docs/api/java/lang/IllegalArgumentException.html
I dont know if it is the best exception to throw, there is short description on this page:
"Thrown to indicate that a method has been passed an illegal or inappropriate argument."
I think that's good enough. 💃